### PR TITLE
Improve Readability of main + Emphasize E-STOP

### PR DESCRIPTION
**Pull request by cmontigel**
Key Changes:
- Combines a significant amount of statements in the configurations present in `__main__.py` for readability
- Add some whitespace between these statements for readability
- Make the text of the `E-STOP` button red and bold to stand out

Additions:
- Creates the style `Quit.TButton` in the `Application` class of `__main__.py`, used by `self.Quit_button`

### CODE REVIEW

File: REVHubInterface/__main__.py [hunk 1]
```diff
@@ -789,6 +519,7 @@ def __init__(self, root):
         style = tkinter.ttk.Style()
         style.configure("Red.Label",   foreground="red")
         style.configure("Green.Label", foreground="green")
+        style.configure("Quit.TButton", foreground='red', font=('TkDefaultFont', 16, 'bold'))
```
Comment (j5155) [inline REVHubInterface/__main__.py:522]: I think the bold text and font size increase is too much here, just making it red would be fine. I found it a little bit distracting and out of place with the large text and bold.

File: REVHubInterface/__main__.py [hunk 2]
```diff
@@ -21,40 +21,27 @@ def __init__(self, root, setAddress):
         self.device_label = tkinter.ttk.Label(self.device_info_frame)
         self.Frame_1 = tkinter.ttk.Frame(self.device_info_frame)
         self.Button_1 = tkinter.ttk.Button(self.Frame_1)
-        self.device_info_frame.config(height=100)
-        self.device_info_frame.config(width=100)
-        self.device_info_frame.grid(column=0)
-        self.device_info_frame.grid(row=0)
-        self.device_info_frame.grid(sticky=(N, S, E, W))
+
+        self.device_info_frame.config(height=100, width=100)
```
Comment (j5155) [inline REVHubInterface/__main__.py:25]: Code formatting changes look fine to me.

**Review by j5155 — CHANGES_REQUESTED** [review]
Looks great overall and I appreciate how quickly you made a PR!
I just want a few tweaks to the text formatting.

**Comment by Iris-TheRainbow** [comment]
looks good. merging.

**Pull request merged**